LayoutTests/imported/w3c/web-platform-tests/cookie-store/cookieStore_set_arguments.https.any.js

```diff
@@ -28,6 +28,11 @@ promise_test(async testCase => {
   assert_equals(cookie.value, 'cookie-value');
 }, 'cookieStore.set with name and value in options');
 
+promise_test(async testCase => {
```

Comment:
please make sure you upstream these changes to WPT afterwards.

Reply:
Will do!

Reply:
Also, are the new tests passing in Chrome?

Reply:
Yes!